### PR TITLE
VZ-6184 Update to Multicluster Troubleshooting documentation

### PR DESCRIPTION
**Pull request by ayaranga**
Updated the multicluster troubleshooting section to re-register the managed cluster in case of the DNS changes on the admin cluster leading to managed cluster status unavailable in the Rancher console.

### CODE REVIEW

**Review by mgianatagh — APPROVED** [review]

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ use the following steps to troubleshoot:
 - Check the multicluster resource's status field on the admin cluster to know what the status of that resource is
   on each managed cluster to which it is targeted.
 
+If you update [DNS]({{< relref "/docs/setup/customizing/dns" >}}) of the admin cluster and notice that the
+managed cluster status is unavailable in the Rancher console along with the error `x509: certificate is valid for
+<rancher new url>, not <rancher old url>` seen in the cattle-cluster-agent (Rancher Agent) logs on the
+managed cluster, then re-register the managed cluster as mentioned [here](#re-register-the-managed-cluster).
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:22]: managed cluster as mentioned [here](#re-register-the-managed-cluster). -> managed cluster, (comma) as described [here](#re-register-the-managed-cluster).

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 2]
```diff
@@ -124,3 +129,46 @@ output:
 ```
 
 The status message contains additional information on the operation's success or failure.
+
+## Re-register the managed cluster
+Perform the below steps to re-register the managed cluster with the admin cluster. The cluster against which to run the
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:134]: the below steps -> the following steps

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 3]
```diff
@@ -124,3 +129,46 @@ output:
...
+command is indicated in each code block.
+#### On the admin cluster
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:136]: Are these steps meant to be followed in order? If so, consider numbering the headings as such:
#### 1. On the admin cluster
#### 2. On the managed cluster
#### 3. On the admin cluster

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Review by rosemarymarano — APPROVED** [review]
Looks good, thank you.

File: content/en/docs/troubleshooting/troubleshooting-multicluster.md [hunk 4]
```diff
@@ -124,3 +129,44 @@ output:
...
+2. On the managed cluster, apply the registration file exported in the previous step, on the managed cluster.
+   ```
+   # On the managed cluster
+   $ kubectl --kubeconfig $KUBECONFIG_MANAGED1 --context $KUBECONTEXT_MANAGED1 \
+       apply -f register_new.yaml
+
+   # Once the command succeeds, you may delete the register_new.yaml file
+   $ rm register_new.yaml
+   ```
+3. On the admin cluster, run `kubectl patch clusters.management.cattle.io` on the admin cluster to trigger redeployment
```
Comment (desagar) [inline content/en/docs/troubleshooting/troubleshooting-multicluster.md:154]: On the admin cluster, run .... on the admin cluster 
(there should probably be only one "on the admin cluster", either beginning or end). 

Similarly above, for "On the managed cluster, apply ...., on the managed cluster".

**Pull request merged**